modules.d/95iscsi/module-setup.sh

```diff
@@ -161,7 +161,7 @@ installkernel() {
     local _funcs='iscsi_register_transport'
 
     instmods bnx2i qla4xxx cxgb3i cxgb4i be2iscsi qedi
-    hostonly="" instmods iscsi_tcp iscsi_ibft crc32c iscsi_boot_sysfs
+    hostonly="" instmods iscsi_tcp iscsi_ibft crc32c iscsi_boot_sysfs 8021q
```

Comment:
> Can you please help to understand why are you also need to change the iscsi module ?

NVMeoF / NBFT and iSCSI  / iBFT boot are very similar concepts. I used the iSCSI code as template for what I did for NVMeoF.  While doing that, I ran into some issues which obviously affect the iSCSI code, too.

> Is this change in the scope of this PR or perhaps should land as a separate PR ?

The iSCSI changes are in separate patches, so I could easily separate them out if you prefer. Still, they are closely related to the nvmf changes, and I figured it might make sense to submit both together.